pages/workspace/workspace.ejs

```diff
@@ -39,11 +65,10 @@
               data-content='<ul class="list-group list-group-flush">
                               <li class="list-group-item p-1"><span class="badge badge-secondary mr-1"><i class="fas fa-fw fa-redo"></i> Refresh</span>Reload this tab at any time, or close and re-open it.</li>
                               <li class="list-group-item p-1"><span class="badge badge-secondary mr-1"><i class="fas fa-fw fa-sync"></i> Reboot</span>Reboot the virtual machine.</li>
-                              <li class="list-group-item p-1"><span class="badge badge-secondary mr-1"><i class="fas fa-fw fa-save"></i> Save</span>Your files are automatically saved.</li>
+                              <li class="list-group-item p-1"><span class="badge badge-secondary mr-1"><i class="fas fa-fw fa-save"></i> Save</span>Your files are automatically saved every <%= workspaceHostFileWatchIntervalSec %> seconds.</li>
```

Comment:
I worry about this a bit because from the user's point of view this may depend on how the image is configured. For example, the Jupyter image defaults to a 120-second autosave interval. Maybe it's better to not specify the time interval?

Reply:
Ok, reverted in 88a3417e61dcf2c70890aafbc777a137721b8b4c. I had added it after losing file changes when testing `Reboot`, so I was again feeling uneasy about the opacity of our autosaving as you're aware.

Reply:
Oh! How did you lose changes? Did you reboot very soon after a change?